### PR TITLE
Babel stuff removed

### PR DESCRIPTION
**Pull request by zouhir**
Please consider looking at #7 comments and let me know if I should do any changes.

### CODE REVIEW

File: tools/cli.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var colors = require('colors');
+
+/**
+ * Checks node version and prints a message before tests start
+ * Latest Nodejs or at least 6.x.x is required to run our ES6 tests
+ * @param {String} nodeVersion node semantic version
+ */
+function checkNode (nodeVersion) {
+  if (Number(nodeVersion.slice('.')[0]) >= 6) {
+    console.log('Your node looks ES6 ready, Yay!'.rainbow);
```
Comment (ericelliott) [inline tools/cli.js]: Delete this condition. This is just noise.

File: tools/cli.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  } else {
+    console.log('These tests requrie Node version > v6.0.0'.red);
```
Comment (ericelliott) [inline tools/cli.js]: This one is fine.

File: tools/cli.js (hunk 1 above)
Comment (zouhir) [inline tools/cli.js]: haha ok done!

File: tools/cli.js (hunk 2 above)
Comment (zouhir) [inline tools/cli.js]: thanks sensei! pushing the changes in a sec.

**Pull request merged**